File: 2013/fall/2/hacker2/generator.c
#define _XOPEN_SOURCE
#include <stdio.h>
#include <unistd.h>

int main(void)
{
    printf("caesar:%s\n", crypt("13", "50"));
    printf("hirschhorn:%s\n", crypt("password", "50"));
    printf("jharvard:%s\n", crypt("crimson", "50"));
    printf("malan:%s\n", crypt("crimson", "HA"));
    printf("milo:%s\n", crypt("1337", "HA"));
    printf("zamyla:%s\n", crypt("1337", "50"));
    return 0;
}

File: 2013/fall/psets/7/pset7/generator.php
#!/bin/env php
<?php

    printf("INSERT INTO users (username, hash) VALUES('%s', '%s');\n", "caesar", crypt("13", '$1$50$'));
    printf("INSERT INTO users (username, hash) VALUES('%s', '%s');\n", "hirschhorn", crypt("password", '$1$50$'));
    printf("INSERT INTO users (username, hash) VALUES('%s', '%s');\n", "jharvard", crypt("crimson", '$1$50$'));
    printf("INSERT INTO users (username, hash) VALUES('%s', '%s');\n", "malan", crypt("crimson", '$1$HA$'));
    printf("INSERT INTO users (username, hash) VALUES('%s', '%s');\n", "milo", crypt("1337", '$1$HA$'));
    printf("INSERT INTO users (username, hash) VALUES('%s', '%s');\n", "skroob", crypt("12345", '$1$50$'));
    printf("INSERT INTO users (username, hash) VALUES('%s', '%s');\n", "zamyla", crypt("1337", '$1$50$'));

?>
